src/v2/ion/uiSchemaTransformer.js

```diff
@@ -51,6 +51,10 @@ const createUISchema = (remediationValue = [], factors = []) => {
   // For simplicity we are assuming that when field itself is a form its only one level deep
   remediationValue = _.chain(remediationValue)
     .map(v => {
+      // return empty array for idps object
+      if (v.relatesTo) {
```

Comment:
hmm, missed this part earlier on.. 
checking `relatesTo` seems having many implications. what about looking for 
- `rel` doesn't exists
- `name` is `redirect` (subject to change when more kinds follow into this categories)

Reply:
Actually, after debugging a bit more, I found that we can remove this check. In responseTransformer we are setting remediation.value only if its present.